library/src/main/assets/google_map.html

```diff
@@ -20,7 +20,11 @@
 var polylines = {};
 var moveTimeout;
 var GeoMarker = null;
+var isChinaMode = false;
+
 function initialize() {
+  AirMapView.onJavaScriptInit();
```

Comment:
do we really need to define this additional method? could we instead utilize `onMapLoaded`, and call `setChinaMode` from the java implementation of `onMapLoaded`?